File: payola/s2js/runtime/src/main/scala/s2js/runtime/scala/Predef.scala
package s2js.runtime.scala

import s2js.runtime.scala.collection.immutable.StringOps

object Predef
{
    def augmentString(x: java.lang.String): StringOps = new StringOps(x)

    def unaugmentString(x: StringOps): java.lang.String = x.repr
}

File: payola/web/server/public/javascripts/runtime/s2js/runtime/scala/Predef.js
goog.provide('scala.Predef');
goog.require('scala.collection.immutable.StringOps');
scala.Predef.augmentString = function(x) {
var self = this;
return new scala.collection.immutable.StringOps(x);
};
scala.Predef.unaugmentString = function(x) {
var self = this;
return x.repr();
};
scala.Predef.__class__ = new s2js.Class('scala.Predef', []);
